addons/packages/packetfence.spec

```diff
@@ -111,7 +111,8 @@ Requires: perl(Class::Accessor)
 Requires: perl(Class::Accessor::Fast::Contained)
 Requires: perl(Class::Data::Inheritable)
 Requires: perl(Class::Gomor)
-Requires: perl(Config::IniFiles) < 2.88
+Requires: perl(Config::IniFiles) >= 2.88
+Requires: perl-Scalar-List-Utils >= 1.42
```

Comment:
I thought you said it wasn't necessary to force the version.

If it is, we should adjust devel...